cssdb.json

```diff
@@ -512,7 +525,7 @@
         "link": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-env-function"
       }
     ],
-    "vendors_implementations": 1
+    "vendors_implementations": 0
```

Comment:
This is still breaking, but it aligns the entry with the plugin and we already special case this plugin.